gc/gc-repository-jdbc/src/main/java/org/projectnessie/gc/contents/jdbc/SqlDmlDdl.java

```diff
@@ -169,13 +170,12 @@ private SqlDmlDdl() {}
           + "    FROM gc_live_set_contents \n"
           + "    WHERE live_set_id = ? AND content_id = ?";
 
-  static final List<String> ALL_CREATES =
-      Collections.unmodifiableList(
-          Arrays.asList(
-              CREATE_LIVE_SETS,
-              CREATE_LIVE_SET_CONTENTS,
-              CREATE_LIVE_SET_LOCATIONS,
-              CREATE_FILE_DELETIONS));
+  static final Map<String, String> ALL_CREATES =
+      Map.of(
+          "gc_live_sets", CREATE_LIVE_SETS,
+          "gc_live_set_contents", CREATE_LIVE_SET_CONTENTS,
+          "gc_live_set_content_locations", CREATE_LIVE_SET_LOCATIONS,
+          "gc_file_deletions", CREATE_FILE_DELETIONS);
 
   static final List<String> ALL_TABLE_NAMES =
```

Comment:
Should this be initialized from `ALL_CREATES.keySet()`? Or removed at all (it's only used in tests IIRC)